community-operators/postgresql/postgres-operator.v4.0.1.clusterserviceversion.yaml

```diff
@@ -139,12 +253,36 @@ spec:
     ### Service
 
     ```
-    oc expose deployment -n operators postgres-operator
+    oc expose deployment -n pgo postgres-operator --type=LoadBalancer
```

Comment:
Avoid use of 'oc' command if this CSV is intended to support upstream-community-operators in addition to community-operators

Reply:
the "community-operator" directory is for OCP/OKD. The "upstream-community-operator" directory is for vanilla Kubernetes versions. Use of 'oc' is appropriate here.

Reply:
The 3.5 upstream-community-operator also includes the 'oc' reference. Just mentioning it here in advance so that it can be removed from the 4.0 upstream operator prior to its submission.

Reply:
currently, the NAMESPACE variable within the container determines what namespace(s) to watch. A comma separated list is supported. See here for details:

https://crunchydata.github.io/postgres-operator/stable/gettingstarted/design/namespace/